internal/repository/prisma/workflow.go

```diff
@@ -397,18 +397,13 @@ func (r *workflowRepository) GetScheduledById(tenantId, scheduleTriggerId string
 }
 
 func (r *workflowRepository) ListWorkflowsForEvent(ctx context.Context, tenantId, eventKey string) ([]*dbsqlc.GetWorkflowVersionForEngineRow, error) {
-	ctx, span := telemetry.NewSpan(ctx, "db-list-workflows-for-event")
-	defer span.End()
+	ctx, span1 := telemetry.NewSpan(ctx, "db-list-workflows-for-event")
+	defer span1.End()
 
-	tx, err := r.pool.Begin(context.Background())
-
-	if err != nil {
-		return nil, err
-	}
-
-	defer deferRollback(context.Background(), r.l, tx.Rollback)
+	ctx, span2 := telemetry.NewSpan(ctx, "db-list-workflows-for-event-query")
+	defer span2.End()
```

Comment:
Isn't this just a duplicate now?

Reply:
Nice catch, I added another `span2.End` later on as well.